### PR TITLE
fix: update distro selection algorithm of `k8s` backend

### PR DESCRIPTION
**Pull request by NayeonKeum**
resolves: #1473 

Error was in `agent/kubernetes/kernel.py:KubernetesCodeRunner.prepare_krunner_env_impl`

Original distro format was `re.search(r"^([a-z]+)\d+\.\d+$", distro)`
image

But when I saw the logs it was somthing like `musllinux_1_2`, `static-gnu` which does not match any of the format.

So I fixed it to getting distro and entrypoint_name, and made it behave same as the docker agent.

- Kubernetes prepare_krunner_env_impl

https://github.com/lablup/backend.ai/blob/ff1e00c90413d643b48c61aa7be8cf4b9d25554f/src/ai/backend/agent/kubernetes/kernel.py#L362-L384

- Docker prepare_krunner_env_impl

https://github.com/lablup/backend.ai/blob/ff1e00c90413d643b48c61aa7be8cf4b9d25554f/src/ai/backend/agent/docker/kernel.py#L376-L390

### CODE REVIEW

File: src/ai/backend/agent/kubernetes/kernel.py [hunk 1]
```diff
@@ -359,31 +358,29 @@ async def get_repl_out_addr(self) -> str:
...
             pkg_resources.resource_filename(
-                f"ai.backend.krunner.{distro_name}", f"./krunner-version.{distro}.txt"
+                f"ai.backend.krunner.{entrypoint_name}", f"./krunner-version.{distro}.txt"
             )
         )
         .read_text()
         .strip()
     )
-    krunner_folder_name = f"backendai-krunner.v{current_version}.{distro}"
+    krunner_folder_name = (  # docker는 {arch}.{distro}임
```
Comment (kyujin-cho) [inline src/ai/backend/agent/kubernetes/kernel.py]: If there is no specific reason to do so please write down all the comments in English.

**Review by kyujin-cho — CHANGES_REQUESTED** [review]
So if I'm understanding correctly, the purpose of this PR is to update outdated distro selection algorithm of Kubernetes agent backend, correct? If it is then please update the PR title accordingly. If not, can you tell a little bit more about it?

File: src/ai/backend/agent/kubernetes/kernel.py (hunk 1 above)
Comment (NayeonKeum) [reply src/ai/backend/agent/kubernetes/kernel.py]: Fixed! Sorry about that 😢

**Comment by NayeonKeum** [comment]
> So if I'm understanding correctly, the purpose of this PR is to update outdated distro selection algorithm of Kubernetes agent backend, correct? If it is then please update the PR title accordingly. If not, can you tell a little bit more about it?

Yep your write. I'll change it to "Fix: update outdated distro selection algorithm of Kubernetes agent backend"

**Review by kyujin-cho — APPROVED** [review]

**Pull request merged**